shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1,37 +1,94 @@
 use crate::helpers::ShotoverManager;
-use cassandra_cpp::{stmt, Session};
 use serial_test::serial;
 use test_helpers::docker_compose::DockerCompose;
 
-use crate::cassandra_int_tests::{assert_query_result, cassandra_connection, ResultValue};
+use crate::cassandra_int_tests::cassandra_connection;
 
-fn test_create_keyspace(session: Session) {
-    assert_query_result(
-        &session,
+mod keyspace {
+    use cassandra_cpp::{stmt, Session};
+
+    use crate::cassandra_int_tests::{assert_query_result, ResultValue};
+
+    fn test_create_keyspace(session: &Session) {
+        assert_query_result(
+        session,
```

Comment:
Theres some wonky formatting here, I think rustfmt is confused by the stmt! macro but if we manually fix the formatting here it wont override it.

Reply:
I'm thinking to change `assert_query_result` to accept a `str` for the statement and just use the macro there. Will make all the tests less verbose and fix these weird formatting issues.

Reply:
I uh... dont think that will work because the macro needs a string literal not a string.
I could be wrong though, so double check.

Reply:
oh, nice!